### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by PastushenkoVik**

### CODE REVIEW

File: src/rleCompress.js [hunk 1]
```diff
@@ -18,7 +18,25 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  const tmp = [...source];
```
Comment (danheim) [inline src/rleCompress.js]: как понять что хранится в переменной?

File: src/rleCompress.js [hunk 2]
```diff
@@ -18,7 +18,25 @@
...
+  const result = [tmp[0]];
```
Comment (danheim) [inline src/rleCompress.js]: тоже самое

**Review by danheim — CHANGES_REQUESTED** [review]

File: src/rleCompress.js (hunk 1 above)
Comment (YuriiHoliuk) [inline src/rleCompress.js]: You will not modify source in the next code, this line is unneeded

File: src/rleCompress.js [hunk 3]
```diff
@@ -18,7 +18,25 @@
...
+  let counter = 1;
+
+  for (let i = 1; i < tmp.length; i++) {
```
Comment (YuriiHoliuk) [inline src/rleCompress.js]: will be better to use array methods, but ok for now

File: src/rleCompress.js [hunk 4]
```diff
@@ -18,7 +18,25 @@
...
+    if (tmp[i] === result[0]) {
+      counter++;
+    } else {
+      if (counter !== 1) {
+        result.unshift(counter);
```
Comment (YuriiHoliuk) [inline src/rleCompress.js]: shift, unshift have very low performance

File: src/rleCompress.js (hunk 2 above)
Comment (YuriiHoliuk) [inline src/rleCompress.js]: `compressedParts` - as an example

Will be better to initialize variable with empty array

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**